Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -363,12 +364,25 @@ namespace AzToolsFramework
 
             if (hasUserSelectedValidSourceFile)
             {
-                // Get position (center of viewport). If no viewport is available, (0,0,0) will be used.
-                AZ::Vector3 viewportCenterPosition = AZ::Vector3::CreateZero();
-                EditorRequestBus::BroadcastResult(viewportCenterPosition, &EditorRequestBus::Events::GetWorldPositionAtViewportCenter);
+                AZ::EntityId parentId;
+                AZ::Vector3 position = AZ::Vector3::CreateZero();
+
+                // if one entity is selected, set it as parent
+                EntityIdList selectedEntities;
+                ToolsApplicationRequestBus::BroadcastResult(selectedEntities, &ToolsApplicationRequests::GetSelectedEntities);
+                if (selectedEntities.size() == 1)
+                {
+                    parentId = selectedEntities.front();
+                    AZ::TransformBus::EventResult(position, parentId, &AZ::TransformInterface::GetWorldTranslation);
```

Comment:
After this change, prefabs can be instantiated:
1. under a nested prefab of a level
2. under a child entity of level
3. under a child entity of nested prefab

Try to test that these 3 use cases work fine. Once these prefabs are instantiated, try to do more operations on them like modifying entities inside them, reparenting them, adding/removing entities under them, converting them into nested instances, etc.

Reply:
Note that those actions were already possible before, but only when dragging and dropping a prefab file from the Asset Browser into the Outliner. So they were part of the testing when Instantiate was added.

We should definitely create automated tests though :)

Reply:
@AMZN-daimini there are already tests for instantiating prefabs.

Reply:
@AMZN-mnaumov only at the root level. The other cases srikappa mentioned should be added (over time).